test/atoms/AtomGraph_tests.jl

```diff
@@ -0,0 +1,70 @@
+using Test
+using LightGraphs
+using SimpleWeightedGraphs
+using Serialization
+
+# NB: featurizing graphs is tested in AtomFeature and GraphNodeFeaturization tests
+
+@testset "AtomGraph" begin
+
+    @testset "construct object" begin
+        # build a cute little triangle graph
+        adj = Float32.([0 1 1; 1 0 1; 1 1 0])
+        g = SimpleWeightedGraph(adj)
+
+        # add an element list that doesn't make sense
+        @test_throws AssertionError AtomGraph(g, ["C"])
+
+        # okay, now do it right, start with no features, checking both constructors
+        ag = AtomGraph(g, ["C", "C", "C"])
+        ag2 = AtomGraph(adj, ["C", "C", "C"])
+        @test adjacency_matrix(ag.graph) == adjacency_matrix(ag2.graph)
+
+        # and now from files...
+        # TODO: this
```

Comment:
Isn't building `AtomGraphs` from files supposed to be in `Utils/GraphBuilding_tests.jl`? So if this `TODO` still valid here, or am I missing something?

Reply:
I think this was updated in one of the more recent commits so it's not there anymore...